Firestore/core/src/model/object_value.cc

```diff
@@ -49,6 +53,27 @@ struct MapEntryKeyCompare {
   }
 };
 
+void SortFields(google_firestore_v1_Value& value) {
```

Comment:
Optional: perhaps add a comment to note that only map fields are sorted?

Reply:
Done